api/app_analytics/serializers.py

```diff
@@ -12,6 +16,12 @@ class UsageDataSerializer(serializers.Serializer):
 class UsageDataQuerySerializer(serializers.Serializer):
     project_id = serializers.IntegerField(required=False)
     environment_id = serializers.IntegerField(required=False)
+    period: PERIOD_TYPE | None = serializers.ChoiceField(
```

Comment:
nit: I'm not sure we're able to type-annotate serializer fields this way. Doesn't matter much until we enforce mypy, of course, but I think this annotation is incorrect.

Reply:
Ok I'll remove typing for it. Not a huge deal either way.